README.md

```diff
@@ -23,22 +29,26 @@ composer require mailjet/mailjet-swiftmailer
 
 ```php
 $transport = new MailjetTransport($dispatchEvent, $apiKey, $apiSecret);
-$transport->setClientOptions(['url' => "www.mailjet.com", 'version' => 'v3', 'call' => true]); // optional
+$transport->setClientOptions(['url' => "api.mailjet.com", 'version' => 'v3.1', 'call' => true]); // optional
```

Comment:
Why is it optional? It depends on which Send API version you're willing to choose, right?